packages/amplication-client/src/Resource/ResourceNameField.tsx

```diff
@@ -14,6 +14,8 @@ import "./ResourceNameField.scss";
 type Props = {
   currentResource: models.Resource;
   resourceId: string;
+  hovered: boolean;
```

Comment:
this should be part of the component

Reply:
The issue was that I didn't get the flag while hovering over the banner, but with prop passing I'm able to show the edit icon in the project name while hovering over the banner.